asdf.fish

```diff
@@ -1,13 +1,15 @@
 #!/usr/bin/env fish
 
-set -l asdf_data_dir (dirname (status -f))
+set -l asdf_dir (dirname (status -f))
+set -l asdf_data_dir (
+  if string length -q -- $ASDF_DATA_DIR; echo $ASDF_DATA_DIR;
```

Comment:
Please use `test -n "$ASDF_DATA_DIR"` instead of `string length`

Reply:
done